app/src/main/res/values/dimens.xml

```diff
@@ -587,7 +587,7 @@
   <dimen name="section_title_text_view_layout_margin_start">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_top">28dp</dimen>
   <dimen name="section_title_text_view_layout_margin_end">32dp</dimen>
-
+  
   <!-- AudioLanguageFragment -->
```

Comment:
(Nit change) @mukulasija this extra space can also be removed.

Reply:
Resolving as this change has been done